appIconIndicators.js

```diff
@@ -379,8 +379,11 @@ var RunningIndicatorDots = class DashToDock_RunningIndicatorDots extends Running
             // Apply dominant color if requested
             if (settings.runningIndicatorDominantColor) {
                 let colorPalette = this._dominantColorExtractor._getColorPalette();
+                let fallbackColor = Clutter.color_from_string(settings.customThemeRunningDotsColor)[1];
                 if (colorPalette !== null) {
                     this._bodyColor = Clutter.color_from_string(colorPalette.original)[1];
+                } else {
+                    this._bodyColor = fallbackColor;
```

Comment:
```suggestion
                const [gotFallbackColor, fallbackColor] = Clutter.color_from_string(settings.customThemeRunningDotsColor);
                if (colorPalette !== null) {
                    this._bodyColor = Clutter.color_from_string(colorPalette.original)[1];
                } else if (gotFallbackColor) {
                    this._bodyColor = fallbackColor;
```

Reply:
What should happen when you can't get the fallback color? Is it possible?

Reply:
Well that's what the bindings provide us, and no it won't never be a null value in such case IIRC.